cmake-edit-core/src/main/java/ru/urururu/cmakeedit/core/checker/Builtins.java

```diff
@@ -0,0 +1,132 @@
+package ru.urururu.cmakeedit.core.checker;
+
+import java.util.Arrays;
+import java.util.List;
+
+/**
+ * Created by okutane on 22/08/16.
+ */
+public class Builtins {
+    /** @see <a href='https://cmake.org/cmake/help/v3.0/manual/cmake-commands.7.html#normal-commands'>cmake help</a> */
+    static List<String> NORMAL = Arrays.asList("add_compile_options",
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Rename this field "NORMAL" to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS3008)